react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+export default class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = ({ IMG: 0 });
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(dir) {
+    if (dir === 'left') {
+      if (this.state.IMG === 0) {
+        this.setState({
+          IMG: 4
+        });
+      } else {
+        this.setState(state => ({
+          IMG: state.IMG - 1
+        }));
+      }
+    } else if (dir === 'right') {
+      if (this.state.IMG === 4) {
```

Comment:
Same here, you don't want to hard code the `4`, you should use the `photos` array to determine what the last index will be.